airbyte-integrations/connectors/source-klaviyo/unit_tests/pyproject.toml

```diff
@@ -7,14 +7,17 @@ name = "source-klaviyo-tests"
 version = "2.14.0"
 description = "Unit tests for source-klaviyo"
 authors = ["Airbyte <contact@airbyte.io>"]
+package-mode = false
 
 [tool.poetry.dependencies]
 python = "^3.10,<3.13"
-airbyte-cdk = "^6"
+airbyte-cdk = "6.45.1.post46.dev14423672753"
```

Comment:
TODO: Bump to stable before merge. Pending:

- https://github.com/airbytehq/airbyte-python-cdk/pull/349